tutorials/ui/gui_skinning.rst

```diff
@@ -1,183 +1,237 @@
 .. _doc_gui_skinning:
 
-GUI skinning
-============
-
-Themes
-------
-
-All control nodes are skinned through the :ref:`Theme <class_Theme>`
-resource. Theme contains all the information required to change the
-entire visual styling of all controls. 
-
-A Theme can be applied to any control node in the scene. As a result,
-all children and grand-children controls will use that same theme, too
-(unless another theme is specified further down the tree). If a value is
-not found in a theme, it will be searched in themes higher up in the
-hierarchy, towards the root. If nothing was found, the default theme is
-used. This system allows for flexible overriding of themes in complex
-user interfaces.
-
-.. attention::
-   
-   Don't use the custom theme option in the Project Settings, as there
-   are known bugs with theme propagation. Instead, apply your theme to the
-   root Control node's Theme property. It will propagate to instanced scenes
-   automatically. To get correct theming in the editor for instanced scenes,
-   you can apply the theme resource to the instanced scene's root node as well.
-
-Creating a theme
+Introduction to GUI skinning
+============================
+
+It is essential for a game to provide clear, informative and yet visually
+pleasing user interface to its players. While :ref:`Control <class_Control>`
+nodes come with a decently functional look out of the box, there is always
+a room for uniqueness and case-specific tuning. For this purpose Godot engine
+includes a system for GUI skinning (or theming), which allows you to customize
+the look of every control in your user interface, including your custom controls.
+
+Here is an example of this system in action, which is radically different from
+the default UI theme of the engine:
+
+*a picture of a game with nice-looking UI*
```

Comment:
Uhm, yeah, I forgot that I still need this. Shouldn't prevent proofreading, just don't merge until I sort this out. Need to get a permission from a dev.